chartify.go

```diff
@@ -692,16 +693,36 @@ func (r *Runner) copyToTempDir(path, tempDir, chartVersion string) (string, erro
 }
 
 func (r *Runner) fetchAndUntarUnderDir(chart, tempDir, chartVersion string) (string, error) {
-	command := []string{"fetch", chart, "--untar", "-d", tempDir}
+	ociEnv := map[string]string{
+		"HELM_EXPERIMENTAL_OCI": "1",
```

Comment:
To be clear, we don't need this anymore, since helm 3.8?

https://github.com/helm/helm/issues/8259